### ISSUE TITLE
Support aborting loading procedure

### ISSUE THREAD

**Issue by atteneder** [atteneder/glTFast#177]
It should be possible to abort the loading procedure at any state without any errors or undisposed resources. All memory should be deallocated properly.

**Comment by rYuuk** [comment]
@atteneder Will there be possibility to abort the loading procedure?

**Comment by atteneder** [comment]
@rYuuk Yes, but there's no timeline as to when it'll be added. I'm thinking about adding a cancelation token to all async calls as a start though.

**Comment by atteneder** [comment]
This has been much improved in [6.16.1](https://docs.unity3d.com/Packages/com.unity.cloud.gltfast@6.19/changelog/CHANGELOG.html#6170---2026-03-17). I consider this done. Report new issues if it does not work as expected.
